test/unit/bin-tessel-2.js

```diff
@@ -521,8 +518,7 @@ exports['Tessel (cli: list)'] = {
 
     test.expect(4);
 
-    var keyPath = './FAKE_KEY';
```

Comment:
@rwaldron because setDefaultKey was called:

`controller.list` => `controller.keyHelper` => because of opts.key !== undefined `provision.setDefaultKey(opts.key)`is called
This is necessary, because `opts.key` is `undefined` what is a bad thing for usability about `t2 root`... you would be required to use `--key` in every case.